app/controllers/user.js

```diff
@@ -24,9 +25,25 @@ exports.addUser = (req, res, next) => {
     .catch(next);
 };
 
-exports.deleteAll = (req, res, next) =>
-  User.destroy({
-    truncate: true
+exports.login = (req, res, next) => {
+  logger.info('Searching user...');
+  return User.findOne({
+    where: {
+      email: req.body.email
+    }
   })
-    .then(() => res.status(200).send('Deleted All'))
+    .then(async user => {
+      if (!user) {
+        throw signinError('Email is not registered in the system');
+      }
+
+      const passwordIsFine = await compare(req.body.password, user.password);
+      if (passwordIsFine) {
+        logger.info('Logged in');
+        const token = signJWT(JSON.stringify(user));
+        return res.set('Authorization', `Bearer ${token}`).send('Logged in');
+      }
+      throw signinError('Email or password is incorrect');
+    })
     .catch(next);
+};
```

Comment:
A little comment, we shouldn't mix promises with async/await. You should chose one.